NtupleProducer/python/HiggsTauTauProducer_106X.py

```diff
@@ -23,7 +23,7 @@
 except:
     SCHEME = "None"
 print 'Uncertainty scheme:', SCHEME
-assert SCHEME in ("None", "MadGraph45A", "MadGraph45B", "MadGraph9A", "MadGraph9B", "Powheg9")
+assert SCHEME in ("MadGraph45A", "MadGraph45B", "MadGraph9A", "MadGraph9B", "MadGraph9B_STlepton", "Powheg9")
```

Comment:
Without "None" the assert fails for data, right?

Reply:
You are correct, and I added the option back.
I was concerned that I had to also adapt the conditions in ```plugins/HTauTauNtuplizer.cc``` (maybe adding some ```if (uncertScheme == "None")``` or similar), but upon inspection it seems that:
- the ```theisMC``` flag (aka ```IsMC```) ensures the data samples do not "see" the uncertainty logic computation for PDF, QCD and astrong
- would a MC sample be by mistake assigned the "None" scheme, a ```cms::Exception("InvalidOption")``` exception would be raised